packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.base.tsx

```diff
@@ -17,7 +17,7 @@ export class TeachingBubbleBase extends BaseComponent<ITeachingBubbleProps, ITea
     /**
      * Default calloutProps is deprecated in favor of private _defaultCalloutProps.
      * Remove in next release.
-     * @deprecated
+     * @deprecated In favor of private _defaultCalloutProps.
```

Comment:
Should `_defaultCalloutProps` be surrounded by back-ticks?

Reply:
Thanks for catching this, it's not strictly necessary but consistency is good.